torchdynamo/allowed_functions.py

```diff
@@ -96,6 +96,12 @@ def _disallowed_function_ids():
         obj for obj in torch.__dict__.values() if isinstance(obj, type(torch.float32))
     ]
     remove += dtypes
+    storage = [
+        obj
+        for obj in torch.__dict__.values()
+        if isinstance(obj, type(torch.FloatStorage))
```

Comment:
I'd consider `torch.storage._LegacyStorageMeta`, but I don't feel strongly there.  We aren't guaranteed there will always be a metaclass here.